contracts/src/contracts/Teleporter/AvalancheICTTRouterFixedFees.sol

```diff
@@ -57,6 +69,40 @@ contract AvalancheICTTRouterFixedFees is
         secondaryRelayerFeeBips = secondaryRelayerFeeBips_;
     }
 
+    /// @inheritdoc IAvalancheICTTRouterFixedFees
+    function registerDestinationTokenBridge(
+        address tokenAddress,
+        bytes32 destinationChainID,
+        address bridgeAddress,
+        uint256 requiredGasLimit,
+        bool isMultihop,
+        uint256 minPrimaryRelayerFee,
+        uint256 minSecondaryRelayerFee
+    ) external onlyOwner {
+        if (tokenAddress != address(0) && !tokenAddress.isContract()) {
+            revert AvalancheICTTRouter__TokenAddrNotAContract(tokenAddress);
+        }
+        if (bridgeAddress == address(0)) {
+            revert AvalancheICTTRouter__BridgeAddrNotAContract(bridgeAddress);
+        }
+        if (destinationChainID == routerChainID) {
+            revert AvalancheICTTRouter__SourceChainEqualsDestinationChain(
+                routerChainID, destinationChainID
+            );
+        }
```

Comment:
We should check here that `minSecondaryRelayerFee` is `0` if `isMultihop` is `false. This would simplify the logic below.